components/landing/TheQuickStart/Algorithms.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <section class="algorithms">
+    <div class="algorithms__container">
+      <h3>Get the SDK</h3>
+      <div class="algorithms__section">
+        <CodeSnippet
+          :code-lines="installCode"
+          :code-snippet-title="'TODO: update this'"
```

Comment:
This probably needs to be changed.

Reply:
Fixed in https://github.com/Qiskit/qiskit.org/pull/3020/commits/cb12310751b2b3a79b6bdc00bd5ea5c44be58646